### PR TITLE
Homepage redesign

### PR DESCRIPTION
**Pull request by brenoferreira**
Integrate https://github.com/creativecommons/vocabulary into CC Search.

---

Developer Certificate of Origin Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors. 1 Letterman Drive Suite D4700 San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or

(b) The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or

(c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.

(d) I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.

### CODE REVIEW

**Comment by brenoferreira** [comment]
Screen Shot 2020-01-10 at 15 52 04

File: src/components/HeroSection.vue [hunk 1]
```diff
@@ -1,36 +1,31 @@
 <template>
   <div class="hero">
-    <img class="logo" alt="Logo" src="../assets/cc-logo_large_black.png">
+    <h1>Search for open content</h1>
```
Comment (kgodey) [inline src/components/HeroSection.vue]: please update this to the latest version of the text proposed by @annatuma and @sarahpearson

File: src/components/HeroSection.vue [hunk 2]
```diff
@@ -1,36 +1,31 @@
...
       </div>
-      <home-license-filter />
-    </form>
-    <div class="description">
+      <div class="description">
         <p>
-          Search for free content in the public domain and under Creative Commons licenses.
-          <br />
-          Learn more about CC licenses <a href="https://creativecommons.org/share-your-work/licensing-examples/" target="_blank" rel="noopener">here</a>.
+          All our content is under Creative Commons licenses.
```
Comment (kgodey) [inline src/components/HeroSection.vue]: Could you double check this text? We've got public domain (CC0) stuff too.

File: src/components/HeroSection.vue [hunk 3]
```diff
@@ -75,79 +71,69 @@ export default {
 <style lang="scss" scoped>
 @import '../../node_modules/foundation-sites/scss/foundation';
 
+h1 {
+  font-family: Roboto;
+  font-size: 2.375em;
+  color: #333333;
```
Comment (kgodey) [inline src/components/HeroSection.vue]: should we use variables for colors?

Comment (brenoferreira) [reply src/components/HeroSection.vue]: This isn't standard yet on CC Search. We'll migrate to using Vocabulary's variables

**Pull request merged**